sources/packages/api/src-sql/StudentAccountApplication/Create-student-account-applications.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS sims.student_account_applications(
+  id SERIAL PRIMARY KEY,
+  submitted_data JSONB NOT NULL,
+  user_id INT NOT NULL REFERENCES sims.users(id),
+  submitted_date TIMESTAMP WITH TIME ZONE NOT NULL,
+  assessed_by INT REFERENCES sims.users(id),
+  assessed_date TIMESTAMP WITH TIME ZONE,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
```

Comment:
I see updated_at is NOT NULL and default value is the same as created_at. I was thinking that this column should only get populated when this table gets updated.

Reply:
Following the pattern in place. If needs to change it must happen globally.

Reply:
Oh never mind then. I will resolve.